gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagActionStore.java

```diff
@@ -69,6 +72,10 @@ public DagNodeId getDagNodeId() {
       return new DagNodeId(this.flowGroup, this.flowName,
           Long.parseLong(this.flowExecutionId), this.flowGroup, this.jobName);
     }
+
+    public void setReminder(boolean isReminder) {
```

Comment:
replace with @setter Lombok annotation

Reply:
I actually wouldn't have expected that to be necessary, given the `@Data` and the non-`final` field